data_store.rb

```diff
@@ -0,0 +1,42 @@
+require 'json'
+require_relative 'music_album'
+require_relative 'genre'
+
+module DataStore
+  def load_albums
+    if File.exist?('album.json')
+      JSON.parse(File.read('album.json')).map do |album|
+        MusicAlbum.new(album['name'], album['genre'], album['label'], album['publish_date'], album['on_spotify'])
+      end
+    else
+      []
+    end
+  end
+
+  def load_genres
+    if File.exist?('genre.json')
+      JSON.parse(File.read('genre.json')).map do |genre|
+        Genre.new(genre['name'])
```

Comment:
Should add the parameter of id in genre on line 19
`Genre.new(genre['id'], genre['name'])`